linkerd/admin/src/main/resources/io/buoyant/linkerd/admin/js/dashboard.js

```diff
@@ -11,11 +11,13 @@ $.when(
 ).done(function(overviewStatsRsp, metricsJson) {
   appendOverviewSection();
   var procInfo = ProcInfo();
+  var dashboard = Dashboard();
+
+  var metricsListeners = [procInfo, dashboard];
+  var metricsCollector = MetricsCollector(metricsListeners);
```

Comment:
Do you need the metricsListeners var here?  Could it just be `var metricsCollector = MetricsCollector([procInfo, dashboard]);`?